drivers/sensor/ccs811/Kconfig

```diff
@@ -1,10 +1,36 @@
 # CCS811 Digital Gas sensor configuration options
 
 # Copyright (c) 2018 Linaro Ltd.
+# Copyright (c) 2018 Peter Bigot Consulting, LLC
+#
 # SPDX-License-Identifier: Apache-2.0
 
-config CCS811
+menuconfig CCS811
 	bool "CCS811 Digital Gas Sensor"
 	depends on I2C && HAS_DTS_I2C
 	help
 	  Enable driver for CCS811 Gas sensors.
+
+if CCS811
+
+choice
```

Comment:
It can not be configured from e.g. shieds Kconfig without a choice symbol, please consider to change to `choice CCS811_DRIVE_MODE`

Reply:
Well, not this one.  I'll see if I have time to get to this and test it.

Reply:
done